src/const.h

```diff
@@ -333,8 +333,10 @@ enum TextColor_t : uint8_t {
 	TEXTCOLOR_DARKRED = 108,
 	TEXTCOLOR_LIGHTGREY = 129,
 	TEXTCOLOR_SKYBLUE = 143,
-	TEXTCOLOR_PURPLE = 155,
+	TEXTCOLOR_PURPLE = 154,
```

Comment:
Added back this text color constant since I had used it to hold the value for electric purple (#1703).